pages/linux/ffuf.md

```diff
@@ -0,0 +1,24 @@
+# ffuf
+
+> A fast web fuzzer written in Go.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Directory discovery using a wordlist on a target website colorized and verbose:
+
+`ffuf -w {{/path/to/wordlist}} -u {{https://target/FUZZ}} -c -v`
```

Comment:
If the path to the wordlist doesn't have to be absolute it should be `path/to/wordlist`.

Reply:
addressed in 4fe26e86dfcc04de1115ef613c87693a5e0b7333